src/custom/components/AccountDetails/Transaction/index.tsx

```diff
@@ -63,266 +73,23 @@ function determinePillColour(status: ActivityStatus, type: ActivityType) {
-      content: '▶';
-      margin: 0 5px 0 0;
-      color: ${({ theme }) => theme.text2};
-      font-size: 8px;
-    }
-  }
-
-  > i {
-    word-break: break-all;
-    white-space: break-spaces;
-
-    ${({ theme }) => theme.mediaWidth.upToSmall`
-      font-weight: 600;
-      margin: 4px 0 0 12px;
-    `}
-
-    &.cancelled {
-      text-decoration: line-through;
-    }
-  }
-`
-
-const TransactionStatusText = styled.div`
-  margin: 0 auto 0 16px;
-  display: flex;
-  align-items: center;
-
-  ${({ theme }) => theme.mediaWidth.upToMedium`
-    margin: 0 auto 0 0;
-  `};
-
-  &.copied,
-  &:hover {
-    text-decoration: none;
-  }
-`
-
-const StatusLabelWrapper = styled.div`
-  display: flex;
-  flex-flow: column wrap;
-  flex: 0 1 auto;
-  justify-content: center;
-`
-
-const StatusLabel = styled.div<{ isPending: boolean; isCancelling: boolean }>`
-  height: 28px;
-  width: 100px;
-  ${({ isPending, isCancelling, theme }) => !isCancelling && isPending && `border:  1px solid ${theme.border2};`}
-  color: ${({ color }) => color};
-  position: relative;
-  border-radius: 4px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  font-size: 12px;
-  font-weight: 600;
-  overflow: hidden;
-
-  &::before {
-    content: '';
-    background: ${({ color, isPending, isCancelling }) => (!isCancelling && isPending ? 'transparent' : color)};
-    position: absolute;
-    left: 0;
-    top: 0;
-    height: 100%;
-    width: 100%;
-    border-radius: 4px;
-    opacity: 0.1;
-  }
-
-  ${({ theme, color, isCancelling, isPending }) =>
-    (isCancelling || isPending) &&
-    color &&
-    css`
-      &::after {
-        position: absolute;
-        top: 0;
-        right: 0;
-        bottom: 0;
-        left: 0;
-        transform: translateX(-100%);
-        background-image: linear-gradient(
-          90deg,
-          rgba(255, 255, 255, 0) 0,
-          ${transparentize(0.9, color)} 20%,
-          ${theme.bg2} 60%,
-          rgba(255, 255, 255, 0)
-        );
-        animation: shimmer 2s infinite;
-        content: '';
-      }
-    `}
-
-  @keyframes shimmer {
-    100% {
-      transform: translateX(100%);
-    }
-  }
-
-  > svg {
-    margin: 0 5px 0 0;
-  }
-`
-
-const TransactionWrapper = styled.div`
-  width: 100%;
-  border-radius: 0;
-  font-size: initial;
-  display: flex;
-  margin: 0;
-  padding: 16px;
-  transition: background 0.2s ease-in-out;
-
-  &:hover {
-    background: rgba(217, 232, 239, 0.35);
-  }
-`
-
-const StatusLabelBelow = styled.div<{ isCancelling?: boolean }>`
-  width: 100%;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  font-size: 12px;
-  line-height: 1.1;
-  margin: 7px auto 0;
-  color: ${({ isCancelling, theme }) => (isCancelling ? theme.primary1 : 'inherit')};
-
-  ${LinkStyledButton} {
-    margin: 2px 0;
-  }
-`
-
 interface OrderSummaryType {
   from: string | undefined
   to: string | undefined
   limitPrice: string | undefined
+  executedPrice?: string | undefined
```

Comment:
I think I commented this somewhere else, maybe it was in Anxo's PR.

Can you use `executionPrice` instead?

Reply:
Done.